apps/src/codebridge/FilePreview/HTMLPreview.tsx

```diff
@@ -1,22 +1,34 @@
 import {useCodebridgeContext} from '@codebridge/codebridgeContext';
+import classNames from 'classnames';
 import React, {useEffect, useMemo, useRef, useState} from 'react';
 
 import codebridgeI18n from '@cdo/apps/codebridge/locale';
 import useLifecycleNotifier from '@cdo/apps/lab2/hooks/useLifecycleNotifier';
 import {isPredictResponseSubmitted} from '@cdo/apps/lab2/redux/predictLevelRedux';
-import {LifecycleEvent} from '@cdo/apps/lab2/utils';
+import {getLabViewPageAction, LifecycleEvent} from '@cdo/apps/lab2/utils';
 import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
 import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 
-import {IframeMessageType, DEFAULT_START_HTML_FILE} from './constants';
-import {UrlBar} from './UrlBar';
+import {
+  IframeMessageType,
+  PreviewViewMode,
+  DEFAULT_START_HTML_FILE,
+} from './constants';
+import {HTMLPreviewHeader} from './HTMLPreviewHeader';
 
 import moduleStyles from './styles/html-preview.module.scss';
 
 const URL_CHANGE_DELAY_MS = 300;
 const SOURCE_CHANGE_DELAY_MS = 500;
 
-export const HTMLPreview = () => {
+export const HTMLPreview: React.FC = () => {
+  const pageAction = getLabViewPageAction();
+  let iframeHeightClass = 'levelViewPreviewIframeHeight';
```

Comment:
should these be css module variables? This might work locally but I think since these are css module classes it won't work on other environments.

Reply:
Also should the logic to define `iframeHeightClass` be wrapped in a `useMemo`?

Reply:
Oh nice catch! Didn't realize strong these in a string variable would not work in other envs. Thanks! Will update.